authors/templates/password_reset/password_reset_email.txt

```diff
@@ -0,0 +1,3 @@
+Dear {{username}},
```

Comment:
Is it necessary to have the message as both a txt file and an html one. Which are we using?

Reply:
We use both because it provides a better consideration for the user's email client preferences. Also because spam filters tend to red-flag more HTML-only emails due to the nature of spam emails.